index.html

```diff
@@ -68,7 +68,78 @@ <h3>Multi-Post Stories</h3>
+                <button type="button" class="job-button">Select Project</button>
+            </div>
+            <div class="job1">
+                <h2>Profesional Art Printing Data</h2>
+                <div class="info-data"><p>A daily section of daily private personalised reads; no accounts or sign ups required ha been the industry standups</p></div>
+                <div class="grid-container2">
+                    <div class="item2">html</div>
+                    <div class="item3">bootstrap</div>
+                    <div class="item3">Ruby</div>   
+                </div>
+                <button type="button" class="job-button">Select Project</button>
+            </div>
+        </section>
+        <section>
+            <div class="me">
+                <div class="about-me">
+                    <h1>About me</h1>
+                </div>
+                <p class="about-info">Hi I am a software developer I can help you build a product,feature or website.Look through some of my work and experience! if you like what you see and have a project you need coded dont hesitate to contact me</p>
+                <button type="button" class="about-button">Get My Resume</button>
+                <div class="deco">
+                    <img class="wall" src="/assets/vectors/Vector-3.png" alt=""/>
+                    <img class="try" src="/assets/vectors/Vector-2.png" alt=""/>
+                    <img class="vector4" src="/assets/vectors/Vector-4.png" alt=""/>
+                    <img class="straight" src="/assets/vectors/Vectorin4.png" alt=""/>
+                </div>
+                <div class="grid-container4">
+                    <div class="heading"><div class="box-rotate"></div><h5>Languages</h5></div>
+                    <div class="javascript">Javascript</div>
+                    <div class="Ruby">Ruby</div>  
+                    <div class="html">Html</div>
+                    <div class="css">Css</div>   
+                </div>
+                <div class="grid-container4">
+                    <div class="heading">
+                        <div class="box"></div>
+                        <h5>Frameworks</h5></div>
+                    <div class="bootstrap">Bootstrap</div>
+                    <div class="Ruby-rails">Ruby on Rails</div>  
+                    <div class="RSPec">RSPec</div>
+                    <div class="capy">Capy Bara</div>   
+                    <div class="salenium">Salenium</div> 
+                </div>
+                <div class="grid-container4">
+                    <div class="heading skills"><div class="dot"></div>
+                    <h5 class="skill">Skills</h5></div>
+                    <div class="codegit">Cogegitt</div>
+                    <div class="github">Github</div>  
+                    <div class="codepen">Codepen</div>
+                    <div class="gitlab">Gitlab</div>   
+                    <div class="terminal">Terminal</div> 
+                </div>
+            </div>
```

Comment:
Please use `ul` and `li` for categories in the "About me" section as was required. instead of your `div` use `li` and remember your `li` should be a child of `ul` read more about tags [here](https://www.w3schools.com/html/html_lists_unordered.asp)